app/renderer/index.html

```diff
@@ -5,25 +5,11 @@
     <title>Appium</title>
     <link rel="stylesheet" href="../../node_modules/font-awesome/css/font-awesome.min.css" />
     <link rel="stylesheet" href="../../node_modules/antd/dist/antd.css" />
-    <script>
-      (function() {
-        if (!process.env.HOT) {
-          const link = document.createElement('link');
-          link.rel = 'stylesheet';
-          link.href = '../../dist/style.css';
-          document.write(link.outerHTML);
-        }
-      }());
-    </script>
+    <link rel="stylesheet" href="styles/app.global.css" />
+    <link rel="stylesheet" href="styles/github-gist-theme.global.css" />
   </head>
   <body>
     <div id="root"></div>
-    <script>
-      (function() {
-        const script = document.createElement('script');
-        script.src = (process.env.HOT) ? 'http://localhost:3000/dist/bundle.js' : '../../dist/bundle.js';
-        document.write(script.outerHTML);
-      }());
-    </script>
+    <script type="text/javascript" src="index.js"></script>
```

Comment:
`/renderer/index.html` is now the entry point and Parcel looks for `index.js` as the JS entrypoint for this script tag and builds at as `<script type="text/javascript" src="renderer.<cache-bust>.js"></script>`